pkgs/development/python-modules/hcloud/default.nix

```diff
@@ -30,6 +31,8 @@ buildPythonPackage rec {
     pytest --ignore=tests/integration
   '';
 
+  passthru.updateScript = nix-update-script { attrPath = "python3Packages.hcloud"; };
```

Comment:
This is not really needed as Python module can be updated with `nix-update`.

Reply:
Removed, thanks.